content/issues/index.md

```diff
@@ -1,76 +1,6 @@
-
-* ![](/images/icons/eye.png) Display data in browser "eye icon"
-* ![](/images/icons/HistoryInfo.png) Job Details and run Info
-* ![](/images/icons/arrow-circle.png) Run this job again or examine original submitted form (filled in)
-* ![](/images/icons/bug.png) Review and optionally submit a bug report
-
-1. How to share a History or Workflow: [Options/Share or Publish](/learn/share/)
-1. If a Workflow was used, please include a share link to the workflow in the bug report's form comments or the direct email.
-1. If still not enough information, the problem can be posted to the [Galaxy Help](https://help.galaxyproject.org/) site (public) or the bug report can be sent in (private).
-1. When submitting a bug report, you'll get cc'd copy of the full report with more details that may help, plus we will review unless you state in the comments that the intention is to see the full report (you can always reply later and request a review).
-1. Should you resolve the problem before hearing back from us, or through other resources/your own testing, a reply email/post stating that is also appreciated.
-1. Please leave all of the datasets undeleted in your history. This includes input and output datasets used in the analysis thread leading up to the problem. Red error datasets are not applied to your account's usage quota. Inputs are often needed to reproduce/troubleshoot a problem.
-1. If you first posted the problem at [Galaxy Help](https://help.galaxyproject.org/), please include a link to that post in the bug/email comments. This helps us to link the two and provide the best feedback.
-1. Please avoid emailing individual team members directly about problems. Use the support channels so both we and the community can help you.
-
-## Bug reporting
-
-If you are cetain that a problem is a software bug, please create a GitHub [issue](https://github.com/galaxyproject/galaxy/issues). If you are not sure, issues can be vetted by following the help for **Usage problem reporting** above.
-
-### TIPS
-
-When creating an issue or sharing details about an error that does not include a shared history (through a link or bug report), be sure to include:
-
-1. Where you are using Galaxy ([Main](/main/), local, docker, or cloud instance).
-1. Bug reports from [Test](/test/) are generally not sent in. We really do test here and the configuration changes frequently.
-1. Galaxy version (usually can be obtained at https://galaxy.example/api/version)
-1. The date/time the bug was detected
-1. Exact steps to reproduce the issue
-1. What troubleshooting steps (if any) you have tested out
+[Reporting usage problems, security issues and bugs](https://training.galaxyproject.org/training-material/faqs/galaxy/#reporting-usage-problems-security-issues-and-bugs) 
```

Comment:
```suggestion
# [Galaxy Issue Reporting and Feature Requests](https://training.galaxyproject.org/training-material/faqs/galaxy/#reporting-usage-problems-security-issues-and-bugs) 
```
let's keep the titles so that the links are not broken